WeatherMood/Utilities/Styles.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Styles.swift
+//  WeatherMood
+//
+//  Created by 이지원 on 2021/07/26.
+//  Reference: https://github.com/kickstarter/ios-oss/blob/main/Library/Styles/BaseStyles.swift
+
+import UIKit
+
+enum Styles {
+    static let cornerRadius: CGFloat = 15.0
+
```

Comment:
```suggestion
    /// 4.0를 곱한만큼 CGFloat값을 return한다.
```
`Styles`로 enum을 만든건 좋았지만, 
`grid` function안에서 `count`를 넣으면 얼마만큼의 `CGFloat`이 나타나는지는 직접 코드를 봐야알 수 있으므로,
코드까지 안보고 어떤값이 곱해지는지 미리 알수있으면 좋겠어요.  ( `Styles.grid`만 쳐도 자동완성으로 주석도 보이니까요. ) 
다른 개발자분이 보실때도 편한 방법으로 고민해주시면 좋겠습니당

Reply:
`Styles.grid(_:)`는 제가 임의로 사용하기 위해서 만들었습니다. 그래서 디자인 가이드가 나오면 해당 메서드를 제거하고, 따로 관리하려고 해서 주석은 추가하지 않았었습니다. 일단 이건 제안해주신 부분 받을게요!